### DISCUSSION TITLE
🚀 Menu Manager - duplicate Menu Set

### DISCUSSION THREAD

**Discussion by ic360-com** [ExpressionEngine/ExpressionEngine#4214] [Feature Requests]
### Suggest an idea for ExpressionEngine

To save time creating different Menu Sets for different Roles, it would be awesome if you could have a Duplicate function, rather than having to create a new one from scratch each time.

**Comment by intoeetive** [comment]
Excellent idea! Check out https://github.com/ExpressionEngine/ExpressionEngine/pull/4222 if you want to use this now and not wait for release
